README.md

```diff
@@ -47,6 +48,9 @@ module "test_bucket" {
     * Label `public` determines if bucket content should be publicly available. IAM rule granting `AllUsers` the role `roles/storage.legacyObjectReader`, enabling public access to objects in the bucket but preventing public listing of bucket content. Default is `"no"`
     * Label `env` can be set to `sandbox` or `production`. If set to `sandbox` suffix will be added to name.
     * Label `tribe` can be set to (`ancillaries`|`autobooking`|`bass`|`bi`|`booking`|`cs-systems`|`data-acquisition`|`finance`|`php`|`platform`|`reservations`|`search`|`tequila`)
+    * Label `active` can be `yes` and `no`. Bucket are not deleted, so use this to signal the bucket is not used.
+    * Label `bill_path` is used to attribute the costs of the bucket to the service. Please use `__` to separate`namespace` and `service`
```

Comment:
1. Not `service` but either `deployment` or `workload` (since that's the proper k8s terminology)
2. Maybe we could mention that only the `namespace` part needs to be there if it's set explicitly. The `deployment/workload` is optional, since we don't break it down per workload yet anyway.